pkg/operation/botanist/component/kubeapiserver/testdata/monitoring_alertingrules.yaml

```
@@ -26,7 +26,7 @@ tests:
   - series: 'apiserver_request_duration_seconds_bucket{verb="POST", le="3"}'
     values: '0+0x62'
   # KubeApiServerTooManyAuditlogFailures
-  - series: 'apiserver_audit_error_total{plugin="webhook", job="kube-apiserver"}'
+  - series: 'apiserver_audit_error_total{plugin!="log", job="kube-apiserver"}'
```

```suggestion
  - series: 'apiserver_audit_error_total{plugin="webhook", job="kube-apiserver"}'
```

This expression says that if there are 32 datapoints of the `apiserver_audit_error_total{plugin="webhook", job="kube-apiserver"}` series with a value of 1, 2, 3, ... then we expect the alert to fire.

So for this change, we don't have to adjust this unit test: the test data `plugin="webhook"` is good to test the expression `plugin!="log"`. For even better test coverage, we could add test data with different plugin values, but I think these test are more like smoke tests so it is sufficient to have only a single test data series.

---

Nice! Thanks for the explanation and the fix!